packages/renderer/src/components/templates/NavbarLayout/index.tsx

```diff
@@ -14,15 +14,27 @@
 //  limitations under the License.
 // =============================================================================
 
+import type { ReactNode } from "react";
+import type { DefaultProps } from "@mantine/core";
 import { Box, createStyles, Group } from "@mantine/core";
-import type { FC } from "react";
 import { Sidebar } from "@organisms/Sidebar";
 
-export const NavbarLayout: FC = (props) => {
-  const { children } = props;
-  const { classes } = useStyles();
+interface NavbarLayoutProps extends DefaultProps {
+  children: ReactNode;
+}
+export const NavbarLayout = ({
```

Comment:
Use `FC` and remove `  children: ReactNode`

Reply:
`NavbarLayoutProps ` extends to `DefaultProps`, so `NavbarLayout ` has more than `children` prop.